docs/docs/04-standard-library/compatibility/compatibility.json

```diff
@@ -1789,6 +1865,106 @@
       "awscdk": {
         "implemented": false
       }
+    },
+    "tryGet": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    },
+    "upsert": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    }
+  },
+  "Endpoint": {
+    "url": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    }
+  },
+  "DynamodbTable": {
+    "putItem": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    },
+    "query": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    },
+    "getItem": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    },
+    "transactWriteItems": {
+      "sim": {
+        "implemented": true
+      },
+      "tf-aws": {
+        "implemented": true
+      }
+    }
+  },
+  "State": {
```

Comment:
nit: State is a simulator-specific resource (not really a cloud primitive) so I think it would be okay to exclude it - up to your discretion